src/ui/playlist/SoundVisualizerIcon.js

```diff
@@ -7,13 +7,16 @@
 import {ul, li, div} from '@cycle/dom'
 import * as S from '../../Utils/StyleUtils'
 
-export default div({
-  className: 'fade-in',
-  style: {
-    ...S.absolute(),
-    ...S.block(50),
-    backgroundColor: 'rgba(255, 255, 255, 0.8)'
-  }
-}, [ul('.playing-animation', {style: {...S.size(17)}}, [
-  li(), li(), li()
-])])
+export default (isAnimating) => {
```

Comment:
Create two versions of this view. One which represents PlayingIcon and the other that represents SelectedIcon. The playing icon should be the animated one.